### ISSUE TITLE
Link to starter uses POST instead of GET

### ISSUE THREAD

**Issue by paulroemer** [vaadin-learning-center/learning-content#118]
Check https://github.com/vaadin-learning-center/learning-content/edit/development/tutorials/vaadin-quick-start/content.adoc line 31.

**Comment by marcushellberg** [comment]
Yes. That was the only way to get it working without support for `GET` methods to the API.
